pages/how-to-configure.md

```diff
@@ -4,9 +4,9 @@
 
 ## How to Configure
 
-💡 The tool is compatible with the SAP Code Inspector framework (`SCI` / `SCII`).
+💡 The tool is compatible with the SAP Code Inspector framework (`SCI` / `SCII`) and you can launch it using your own SCI/SCII variant(s). Alternatively, you may also use the tool by means of defining Profiles as described bellow.
```

Comment:
@estevao-schultz-neto-SAP: it looks redundant as we mentioned the transactions twice.